src/modules/helpchan.ts

```diff
@@ -277,6 +277,48 @@ export class HelpChanModule extends Module {
 		await msg.delete({ reason: 'Pin system message' });
 	}
 
+	@command({
+		description: 'Pings a helper in a help-channel',
+		aliases: ['helpers'],
```

Comment:
```suggestion
		aliases: ['helpers'],
		inhibitors: [CommonInhibitors.guildsOnly]
```

Reply:
Wouldn't that be covered by the `msg.channel.parentID !== categories.ongoing` check?

Reply:
It *is* a nitpick (: